### PR TITLE
Added Voltage to Web deployment

### PR DESCRIPTION
**Pull request by Bangalisch**
See #1038  for more information.
All these topics covered in this update.
Checking first results on links in first commit.

### CODE REVIEW

File: docs/Deployment/voltagecloud.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+This is when we will start working towards our **BTCPay server node**. 
+
+The top right corner of your dashboard will allow you to switch between `Nodes` and `Flow`.
+Click on `Nodes` as we want a BTCPay server node and create a new node. 
+
+![Voltage Cloud Node](../img/voltage/whatnode.jpg "Voltage Cloud Node")
+
+## 4. BTCPay server node.
+
+Now we are looking at creating a BTCPay server node.
```
Comment (pavlenex) [inline docs/Deployment/voltagecloud.md:65]:
```suggestion
Now we are looking at creating a BTCPay Server node. 
```

File: docs/Deployment/voltagecloud.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+You now have to set a `Store name` for the main store of your BTCpay server. 
+The choice to connect it to your Voltage node and pick the node you created earlier (the LND node)
+And lock this in by typing the password set for the LND node. 
+
+![Voltage Cloud BTCPay server Node](../img/voltage/deploybtcpay.jpg "Voltage Cloud Cloud BTCPay server Node")
+
+## 5. Finnish initial deployment. 
+
+With all the steps taken above and following along with the video above,
+You are now at the point where voltage shows you the BTCPay server Information.
```
Comment (pavlenex) [inline docs/Deployment/voltagecloud.md:78]:
```suggestion
With all the steps taken above and following along with the video above, you are now at the point where Voltage shows you the BTCPay server information. 
```

File: docs/Deployment/voltagecloud.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+![Voltage Cloud BTCPay server Node](../img/voltage/deploybtcpay.jpg "Voltage Cloud Cloud BTCPay server Node")
+
+## 5. Finnish initial deployment. 
+
+With all the steps taken above and following along with the video above,
+You are now at the point where voltage shows you the BTCPay server Information. 
+
+* URL for BTCPay server access
+* Username for the BTCPay server store.
```
Comment (pavlenex) [inline docs/Deployment/voltagecloud.md:81]:
```suggestion
* Username for the BTCPay Server's store. 
```

File: docs/Deployment/voltagecloud.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+![Voltage Cloud BTCPay server Node](../img/voltage/deploybtcpay.jpg "Voltage Cloud Cloud BTCPay server Node")
+
+## 5. Finnish initial deployment. 
+
+With all the steps taken above and following along with the video above,
+You are now at the point where voltage shows you the BTCPay server Information. 
+
+* URL for BTCPay server access
+* Username for the BTCPay server store. 
+* default password
```
Comment (pavlenex) [inline docs/Deployment/voltagecloud.md:82]:
```suggestion
* Default password
```

File: docs/Deployment/voltagecloud.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+* Attached Node Name 
+* Creation date of the node
+* Expiration date
+* Purchase status 
+
+A button for easy access to your BTCPay server dashboard and the button to Delete the instance also became available to you.
```
Comment (pavlenex) [inline docs/Deployment/voltagecloud.md:88]:
```suggestion
A button for easy access to your BTCPay Server dashboard and the button to delete the instance also became available to you. 
```

File: docs/Deployment/voltagecloud.md [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+* URL for BTCPay server access
+* Username for the BTCPay server store. 
+* default password
+* Attached Node Name 
+* Creation date of the node
+* Expiration date
+* Purchase status 
+
+A button for easy access to your BTCPay server dashboard and the button to Delete the instance also became available to you. 
+Now click the button to go to the BTCPay server dashboard, and you are ready to start operating as a merchant on Bitcoin!
```
Comment (pavlenex) [inline docs/Deployment/voltagecloud.md:89]:
```suggestion
Now click the button to go to your BTCPay server dashboard, and you are ready to start using BTCPay! 
```

File: docs/Deployment/voltagecloud.md [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+* default password
+* Attached Node Name 
+* Creation date of the node
+* Expiration date
+* Purchase status 
+
+A button for easy access to your BTCPay server dashboard and the button to Delete the instance also became available to you. 
+Now click the button to go to the BTCPay server dashboard, and you are ready to start operating as a merchant on Bitcoin! 
+
+## 6. Welcome to your BTCPay server dashboard.
```
Comment (pavlenex) [inline docs/Deployment/voltagecloud.md:91]:
```suggestion
## 6. Welcome to your BTCPay Server dashboard
```

File: docs/Deployment/voltagecloud.md [hunk 8]
```diff
@@ -0,0 +1,99 @@
...
+* Creation date of the node
+* Expiration date
+* Purchase status 
+
+A button for easy access to your BTCPay server dashboard and the button to Delete the instance also became available to you. 
+Now click the button to go to the BTCPay server dashboard, and you are ready to start operating as a merchant on Bitcoin! 
+
+## 6. Welcome to your BTCPay server dashboard. 
+
+You are now inside your new BTCPay server.
```
Comment (pavlenex) [inline docs/Deployment/voltagecloud.md:93]:
```suggestion
You are now inside your new BTCPay Server. 
```

File: docs/Deployment/voltagecloud.md [hunk 9]
```diff
@@ -0,0 +1,99 @@
...
+* Purchase status 
+
+A button for easy access to your BTCPay server dashboard and the button to Delete the instance also became available to you. 
+Now click the button to go to the BTCPay server dashboard, and you are ready to start operating as a merchant on Bitcoin! 
+
+## 6. Welcome to your BTCPay server dashboard. 
+
+You are now inside your new BTCPay server. 
+It shows in the dashboard that you've created a store and that Lightning Network has been set up. 
+The Bitcoin Wallet is still missing. You can follow along in [this wallet setup guide](../WalletSetup.md)
```
Comment (pavlenex) [inline docs/Deployment/voltagecloud.md:95]:
```suggestion
The bitcoin wallet is still missing. You can follow along in [this wallet setup guide](../WalletSetup.md)
```

File: docs/Deployment/webdeployment.md [hunk 10]
```diff
@@ -0,0 +1,32 @@
+# Introduction to Web Deployments.
+
+We will take you through possible `Web Deployments` on this page.
+We will keep updating the list as more providers offer BTCPay server as a service.
```
Comment (pavlenex) [inline docs/Deployment/webdeployment.md:4]: We should explain what web deployment is here "services who offer managed BTCPay Server hosting..."

**Review by pavlenex — COMMENTED** [review]
Small corrections and a few comments regarding the structure, mostly web deployment page needs an explanations on what is it and how it works. Good work putting this one through!

File: docs/Deployment/voltagecloud.md [hunk 11]
```diff
@@ -0,0 +1,99 @@
...
+
+## 2. What node suits you? 
+
+The dashboard asks us what to do. 
+In this section, we go into the `Nodes` tab and leave `Flow` for now. 
+By clicking `Nodes`, we are now prompted to **Create Node**
+
+For running the BTCPay server with Lightning-enabled, we will have to create two nodes. 
+First, you have to create the Lightning Node, click on LND button. 
+After that, it makes your choice between three options of nodes.
```
[the remark this answers is not in the document]
Comment (Bangalisch) [reply docs/Deployment/voltagecloud.md:32]: Don't know, will figure this out at Voltage. 
After doing the research, I'll add it to the list.

File: docs/Deployment/webdeployment.md (hunk 10 above)
Comment (Bangalisch) [reply docs/Deployment/webdeployment.md:4]: Agree. Will write a little more to explain. 
Thanks.

**Review by pavlenex — APPROVED** [review]
Good work!

**Pull request merged**